### PR TITLE
on hover show user name + image

### PR DESCRIPTION
**Pull request by StanislavShevchenko**
# Description

1 - Created tooltip with user info
2 - Added new columns in tx table and added dynamic sort
3 - Added paymentLinkContract condition for from tx address

About #4 

# How Has This Been Tested?

1 - Hover on wallet address in tables wallet and transactions 
2 - Lookin transactions table in header 

# Checklist:
- [x] PR title matches follow: (Feature|Bug|Chore) Task Name
- [x] My code follows the style guidelines of this project
- [x] I have followed all the instructions described in the initial task (check Definitions of Done)
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have added reference to a related issue in the repository
- [x] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [ ] I have added screenshots related to my pull request ( for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [ ] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

File: client/src/lib/gun/gundb.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import 'gun/lib/rindexed'
+
+let gunDb
+
+const initGunDB = () => {
+
+    if (!gunDb) {
+        gunDb = Gun({
+            localStorage: false,
+            peers: ['https://etorogun-prod.herokuapp.com/gun']
```
Comment (AlexeyKosinski) [inline client/src/lib/gun/gundb.js]: add URL to ENV and config

**Review by AlexeyKosinski — CHANGES_REQUESTED** [review]
also please add @gooddollar/gun-appendonly to package.json

**Review by AlexeyKosinski — COMMENTED** [review]
please change:
- invert sort icon
- reduce size of addresses in tables
- remove user full name: in tooltip
- change tooltip orientation to top
- fix numbers in balances table (2-11 to 1-10)
- change ui in middle screen size to prevent horizontal scroll

**Comment by StanislavShevchenko** [comment]
> please change:
> 
> * invert sort icon
> * reduce size of addresses in tables
> * remove user full name: in tooltip
> * change tooltip orientation to top
> * fix numbers in balances table (2-11 to 1-10)
> * change ui in middle screen size to prevent horizontal scroll

done in https://github.com/GoodDollar/GoodDashboard/pull/8/commits/486abd86b9c74307aef37603d34567c603129ab1

**Review by AlexeyKosinski — CHANGES_REQUESTED** [review]
@StanislavShevchenko please fix ui in table
Screenshot 2020-01-10 at 18 23 24
make it inline

**Comment by StanislavShevchenko** [comment]
> @StanislavShevchenko please fix ui in table
> Screenshot 2020-01-10 at 18 23 24
> make it inline

done https://github.com/GoodDollar/GoodDashboard/pull/8/commits/67e66ebe8e3dc5f08cd8a2c18cbf8b0118a97413
image

**Review by AlexeyKosinski — CHANGES_REQUESTED** [review]
Screenshot 2020-01-16 at 10 10 59
Screenshot 2020-01-16 at 10 11 15
please change ui to prevent horizontal scroll. make sort arrow inline on all screen sizes

**Review by AlexeyKosinski — APPROVED** [review]

File: server/src/providers/blockchain.ts [hunk 2]
```diff
@@ -396,7 +407,7 @@ export class blockchain {
       })
 
       // log.debug("Event:", { fromAddr, toAddr, event });
-      if (this.isClientWallet(fromAddr)) {
+      if (this.isClientWallet(fromAddr) || this.isPaymentlinkContracts(fromAddr)) {
```
Comment (sirpy) [inline server/src/providers/blockchain.ts]: Why are we including the paymentlink contract?

File: server/src/providers/blockchain.ts [hunk 3]
```diff
@@ -430,7 +441,7 @@ export class blockchain {
         log.trace('Skipping system contracts event', { fromAddr })
       }
 
-      if (this.isClientWallet(toAddr)) {
+      if (this.isClientWallet(toAddr) && (this.isClientWallet(fromAddr) || this.isPaymentlinkContracts(fromAddr))) {
```
Comment (sirpy) [inline server/src/providers/blockchain.ts:444]: What is the reason for this change?

**Review by sirpy — CHANGES_REQUESTED** [review]

File: server/src/providers/blockchain.ts (hunk 3 above)
Comment (AlexeyKosinski) [reply server/src/providers/blockchain.ts:444]: it was task - `change calculation of "in" txs, incoming tx is counted when the "from" is the paymentlink contracts or the from is not a system account`

**Review by AlexeyKosinski — CHANGES_REQUESTED** [review]
after change order in user transactions table user data is not loading. just infinity spinner
Screenshot 2020-01-20 at 14 42 03

**Comment by StanislavShevchenko** [comment]
@AlexeyKosinski. I did it so that the information would load after hovering

**Review by sirpy — CHANGES_REQUESTED** [review]
currently on gooddashboard it seems like after I resort the top TX users the hover stops working

**Comment by AlexeyKosinski** [comment]
> currently on gooddashboard it seems like after I resort the top TX users the hover stops working

I've run deploy dashboard. so now it will work like describe @StanislavShevchenko

**Review by AlexeyKosinski — APPROVED** [review]
